OADP-2157: Updating IBM Power and IBM Z systems section (#73935)
openshift/openshift-docs: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc

OpenShift API for Data Protection (OADP) is platform neutral. The information that follows relates only to {ibm-power-name} and to {ibm-z-name}.

> OADP 1.1.4 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.4 in terms of backup locations for these systems.

Reviewer: Here as well please.

> OADP 1.1.4 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.4 in terms of backup locations for these systems.

Author: ```suggestion
* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.7 in terms of backup locations for these systems.
* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.2.3 in terms of backup locations for these systems.
* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.
```

OpenShift API for Data Protection (OADP) is platform neutral. The information that follows relates only to {ibm-power-name} and to {ibm-z-name}.

> * {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.7 in terms of backup locations for these systems.

Author: ```suggestion
* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.1.7 in terms of backup locations for these systems.
```

> * {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.2.3 in terms of backup locations for these systems.

Author: ```suggestion
* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.
```

> * {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.

Author: ```suggestion
* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12, 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.
```

> * {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.

Author: ```suggestion
* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.3.1 in terms of backup locations for these systems.
```

> * {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.

Author: ```suggestion
* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.
```
